### DISCUSSION TITLE
Does photoprism allow us to reorganize photographs?

### DISCUSSION THREAD

**Discussion by nipunarora** [photoprism/photoprism#1942] [Q&A] [answered]
Does photoprism allow us to re-organize photographs from within the UI. Also if I move files around in different directories indexed by photoprism (outside of photoprism). Will I have to do a complete re-index, or will regular indexing recognize the moved locations? 

My most frequent workflow is to dump the photographs into a single folder, and then after a few months come back and re-organize the photos. This typically happens with iphone camera uploads.

**Comment by nautatva** [comment]
Also, does photoprism detect resolution changes? I migrated from google photos and indexed them. Now I got the original copies for them in higher resolution and would like photoprism to reindex them. Does de-duplication suggest me a better photo?

**Reply by graciousgrey** [reply]
PhotoPrism detects exact duplicates using sha1 hashes. Those hashes are different if you have the same file in different resolutions (those are not exact duplicates). In case your files have sequential names or the same date and time metadata the original and low resolution version might already be stacked (see https://docs.photoprism.app/user-guide/organize/stacks/#for-what-reasons-can-files-be-stacked).
It's planned to automatically stack original and compressed versions: https://github.com/photoprism/photoprism/issues/1182

**Comment by graciousgrey** [comment, marked as answer]
@nipunarora you can't change the physical location of your files using the web ui. If you move files in your originals folder using the filesystem, a normal rescan is sufficient to detect the new path.

**Reply by IchBinGROOT** [reply]
If I move photos in the underlying file system that I manually added metadata to, like selecting faces, will they be automatically recognized in the new photo location?

**Reply by graciousgrey** [reply]
Yes, if you move files and let PhotoPrism re-index the library afterwards, it will recognize the new path. Metadata such as tagged faces or similar are not changed.

**Comment by nipunarora** [comment]
@graciousgrey - thanks for explaining, how do I see the duplicate files detected by photoprism?

**Reply by nipunarora** [reply]
for example if I want to delete duplicates, I'd like to first know where they all are?

**Reply by graciousgrey** [reply]
We have an open ticket to do this in the web UI (https://github.com/photoprism/photoprism/issues/1308). 
For now you find duplicates in the duplicates table in the database.
